cli/remotebazel/remotebazel.go

```diff
@@ -86,6 +86,7 @@ var (
 	runFromBranch           = RemoteFlagset.String("run_from_branch", "", "A GitHub branch to base the remote run off. If unset, the remote workspace will mirror your local workspace.")
 	runFromCommit           = RemoteFlagset.String("run_from_commit", "", "A GitHub commit SHA to base the remote run off. If unset, the remote workspace will mirror your local workspace.")
 	script                  = RemoteFlagset.String("script", "", "Shell code to run remotely instead of a Bazel command.")
+	key                     = RemoteFlagset.String("key", "", "Shell code to run remotely instead of a Bazel command.")
```

Comment:
nit: update help string, also maybe call it `snapshot_key`? or `snapshot`

Reply:
I liked that. Changed to `run_from_snapshot`